cmd/installer/pkg/bootloader/syslinux/syslinux.go

```diff
@@ -35,6 +37,18 @@ const syslinuxLabelTpl = `LABEL {{ .Root }}
 `
 
 const (
+	// BootA is a syslinux label.
+	BootA = "boot-a"
+
+	// BootB is a syslinux label.
+	BootB = "boot-b"
+
+	// SyslinuxLdlinux is the path to ldlinux.sys.
+	SyslinuxLdlinux = constants.BootMountPoint + "/syslinux/ldlinux.sys"
+
+	// SyslinuxConfig is the path to the Syslinux config.
+	SyslinuxConfig = constants.BootMountPoint + "/syslinux/syslinux.cfg"
+
```

Comment:
As we start to modularize, it makes sense to have the syslinux specific constants live in this package.

Reply:
How about a `constants.go` file?  I don't have a strong opinion of pulling out constants to a separate file, but since we _currently_ pull them to a separate _package_, I wonder if we couldn't retain the value in that way.

Reply:
👍 works for me.